packs/feat-effects/effect-come-and-get-me.json

```
@@ -0,0 +1,56 @@
+{
+    "_id": "k29Ao7wxjPVqxuez",
+    "img": "icons/skills/targeting/crosshair-pointed-orange.webp",
+    "name": "Effect: Come and Get Me",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Come and Get Me]</p>\n<p>Until your rage ends, you are @UUID[Compendium.pf2e.conditionitems.Item.Off-Guard], and damage rolls against you gain a +2 circumstance bonus. If a creature hits you, that creature is @UUID[Compendium.pf2e.conditionitems.Item.Off-Guard] to you until the end of your next turn. If you hit it before the end of your next turn, you gain temporary @Damage[(@actor.level)[healing]]{Hit Points equal to your Constitution modifier}, or @Damage[(2*(@actor.level))[healing]]{double} that on a critical hit. These temporary Hit Points last until the end of your rage.</p>"
```

```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Come and Get Me]</p>\n<p>You are @UUID[Compendium.pf2e.conditionitems.Item.Off-Guard], and damage rolls against you gain a +2 circumstance bonus.</p>"
```
Effects should only say exactly what they automate.

---

Add a note for off guard on damage received and another note linking to the temp HP effect on hitting an off guard creature